components/portal/src/components/image/index.js

```diff
@@ -353,7 +353,7 @@ class Image extends React.Component {
                             </Typography>
                         </div>
                         {
-                            globalState.get(StateHolder.USER).roles.includes(Constants.Permission.PUSH)
+                            (imageData.userRole === Constants.Permission.PUSH)
```

Comment:
Need to check whether the userRole is push or admin both

Reply:
fixed